cv_next/components/pages/deactivated.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from "next/link"
+import Image from "next/image"
+import { buttonVariants } from "@/components/ui/button"
+import { cn } from "@/lib/utils"
+import { inactiveHeader } from "@/config/contents"
+
+export default function HeroHeader() {
```

Comment:
1. Name.. 
2. Perhaps split this down to smaller components (can be local in the same file)
3. Please do not export `default` on non-page components, only named exports